### PR TITLE
Generate Dynamic Phrases

### PR DESCRIPTION
**Pull request by awburgard**

### CODE REVIEW

File: src/utils/translations/processDataForTranslations.ts [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+      } catch (error) {
+        console.error(`JSON Parsing Error: ${error}`);
+      }
+    }
+
+    if (configVar.dataType === "picklist") {
+      processProperties(configVar.pickList ?? []);
+    }
+
+    // if (
```
Comment (awburgard) [inline src/utils/translations/processDataForTranslations.ts:121]: if we ever want to cover parsing JSON forms this gets us part of the way there

File: src/commands/translations/integration.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const {
+      args: { integration },
+      flags: { "output-file": output },
+    } = await this.parse(TranslationsCommand);
+
+    const cwd = process.cwd();
+
+    const result = await gqlRequest<MarketplaceTranslations>({
+      document: GET_MARKETPLACE_INTEGRATIONS_TRANSLATIONS,
+      variables: { id: integration },
```
Comment (taylorreece) [inline src/commands/translations/integration.ts:37]: It doesn't look like `GET_MARKETPLACE_INTEGRATIONS_TRANSLATIONS` has query parameters, so `id` passed in here doesn't have any affect. `translations:integration` and `translations:list` both currently behave the same

File: src/queries.graphql.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { gql } from "./graphql";
+
+export const GET_MARKETPLACE_INTEGRATIONS_TRANSLATIONS = gql`
```
Comment (taylorreece) [inline src/queries.graphql.ts:3]: Probably not the most critical thing for now, but if anyone ever has more than 100 marketplace offerings, we'll need to paginate.

**Review by taylorreece — APPROVED** [review]
I rebased and force pushed to make sure this is up-to-date. Just a couple quick notes.

**Comment by taylorreece** [comment]
Quick note: please update `.gitignore` to ignore `translations_output.json`.

**Comment by taylorreece** [comment]
Latest LGTM!

**Review by bkegley — APPROVED** [review]
This looks good!

**Pull request merged**